resources/views/pages/ticket/[ticket].blade.php

```diff
@@ -0,0 +1,423 @@
+        </div>
+        <div class="flex flex-col gap-y-1">
+            <p role="heading" aria-level="2" class="mb-0.5 text-2xs font-bold">Achievement Information</p>
+                <div class="relative w-full p-2 bg-embed rounded">
+                    <x-ticket.stat-element label="Achievement">{!! achievementAvatar($ticket->achievement, iconSize:16) !!}</x-ticket.stat-element>
+                    <x-ticket.stat-element label="Game">{!! gameAvatar($ticket->achievement->game, iconSize:16) !!}</x-ticket.stat-element>
+                    <x-ticket.stat-element label="Author">{!! userAvatar($ticket->achievement->author, iconSize:16) !!}</x-ticket.stat-element>
+        
+                    @if ($ticket->achievement->type)
+                        <x-ticket.stat-element label="Type">{{ __('achievement-type.' . $ticket->achievement->type) }}</x-ticket.stat-element>
+                    @endif
+
+                    @php $label = TicketState::isOpen($ticket->ReportState) ? 'Other open tickets' : 'Open tickets' @endphp
+                    @if (empty($openTicketLinks))
+                        <x-ticket.stat-element label="{{ $label }}"><span class="text-muted">None</span></x-ticket.stat-element>
+                    @else
+                        <x-ticket.stat-element label="{{ $label }} ({{ count($openTicketLinks) }})">{!! implode(', ', $openTicketLinks) !!}</x-ticket.stat-element>
+                    @endif
+        
+                    @php $label = TicketState::isOpen($ticket->ReportState) ? 'Closed tickets' : 'Other closed tickets' @endphp
+                    @if (empty($closedTicketLinks))
+                        <x-ticket.stat-element label="{{ $label }}"><span class="text-muted">None</span></x-ticket.stat-element>
+                    @else
+                        <x-ticket.stat-element label="{{ $label }} ({{ count($closedTicketLinks) }})">{!! implode(', ', $closedTicketLinks) !!}</x-ticket.stat-element>
+                    @endif
+                </div>
+            </p>
+        </div>
+    </div>
+
+    @if ($permissions >= Permissions::Developer)
+        <div class="mt-2">
+            <div class="flex w-full justify-between border-embed-highlight items-center">
+                <p role="heading" aria-level="2" class="mb-0.5 text-2xs font-bold">
+                    Player history for game
+                </p>
+            </div>
+
+            <div class="relative w-full p-2 bg-embed rounded">
+                <div class="w-full relative flex gap-x-3">
+                @if (empty($userAgentLinks))
+                    <span class="text-muted">No client data available</span>
+                @else
+                    <span>
+                        <span class="font-bold">Clients used:</span>
+                        <span>{!! implode(', ', $userAgentLinks) !!}</span>
+                    </span>
+                @endif
+
+                    <button id="unlockHistoryButton" class="absolute bottom-0 right-0 btn"
```

Comment:
Changing "bottom-0 right-0" to "-top-2 -right-2" will let the button sit flush against the container edge.

Reply:
I like it flush with the expanded stuff more than the outer container.

Current:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/332a2a40-03be-4fe0-b8a5-25ea77ef04c2)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/46c965d0-a05c-475a-8ec6-fb311ec103bc)

Your proposal:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/aaa50acb-0dd6-4ce4-9216-fd4ef7fd5860)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/28f37467-fafe-4d67-971d-da8bcf167885)

Reply:
That reasoning makes sense, though it seems inconsistent with how we use this UI pattern. I'd argue the component design shouldn't change because of some other element somewhere on the page, for these reasons:

1. It makes it more difficult to wrap this up into something reusable, as there's a one-off modification on a page somewhere.
2. It's plausible the "Award Hardcore" button won't be there one day, in which case the shifted position of the "Unlock History" button won't make sense.

I don't feel strongly enough about it to express more concern beyond this, as I can already think of counterarguments to these two points.